src/DB_Command.php

```diff
@@ -789,120 +795,124 @@ public function size( $args, $assoc_args ) {
+					DB_NAME
 				)
 			);
 
 			// Add the database size to the list.
 			$rows[] = array(
-				'Name'  => DB_NAME,
-				'Size'  => strtoupper( $db_bytes ) . $default_unit,
-				);
+				'Name' => DB_NAME,
+				'Size' => strtoupper( $db_bytes ) . $default_unit,
+			);
 		}
 
 		if ( ! empty( $size_format ) || $human_readable ) {
-			foreach( $rows as $index => $row ) {
+			foreach ( $rows as $index => $row ) {
 					// These added WP 4.4.0.
-					if ( ! defined( 'KB_IN_BYTES' ) ) {
-						define( 'KB_IN_BYTES', 1024 );
-					}
-					if ( ! defined( 'MB_IN_BYTES' ) ) {
-						define( 'MB_IN_BYTES', 1024 * KB_IN_BYTES );
-					}
-				        if ( ! defined( 'GB_IN_BYTES' ) ) {
-						define( 'GB_IN_BYTES', 1024 * MB_IN_BYTES );
-					}
-				        if ( ! defined( 'TB_IN_BYTES' ) ) {
-						define( 'TB_IN_BYTES', 1024 * GB_IN_BYTES );
-					}
+				// phpcs:disable WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
+				if ( ! defined( 'KB_IN_BYTES' ) ) {
+					define( 'KB_IN_BYTES', 1024 );
+				}
+				if ( ! defined( 'MB_IN_BYTES' ) ) {
+					define( 'MB_IN_BYTES', 1024 * KB_IN_BYTES );
+				}
+				if ( ! defined( 'GB_IN_BYTES' ) ) {
+					define( 'GB_IN_BYTES', 1024 * MB_IN_BYTES );
+				}
+				if ( ! defined( 'TB_IN_BYTES' ) ) {
+					define( 'TB_IN_BYTES', 1024 * GB_IN_BYTES );
+				}
+				// phpcs:enable
 
-					if ( $human_readable ) {
-						$size_key = floor( log( $row['Size'] ) / log( 1000 ) );
-					    $sizes = array( 'B', 'KB', 'MB', 'GB', 'TB' );
+				if ( $human_readable ) {
+					$size_key = floor( log( $row['Size'] ) / log( 1000 ) );
+					$sizes    = array( 'B', 'KB', 'MB', 'GB', 'TB' );
 
-						$size_format = isset( $sizes[ $size_key ] ) ? $sizes[ $size_key ] : $sizes[ 0 ];
-					}
+					$size_format = isset( $sizes[ $size_key ] ) ? $sizes[ $size_key ] : $sizes[0];
+				}
 
 					// Display the database size as a number.
-					switch( $size_format ) {
-						case 'TB':
-						         $divisor = pow( 1000, 4 );
-							 break;
-
-						case 'GB':
-						         $divisor = pow( 1000, 3 );
-							 break;
-
-						case 'MB':
-						         $divisor = pow( 1000, 2 );
-							 break;
-
-						case 'KB':
-						         $divisor = 1000;
-							 break;
-
-						case 'tb':
-						case 'TiB':
-						         $divisor = TB_IN_BYTES;
-							 break;
-
-						case 'gb':
-						case 'GiB':
-						         $divisor = GB_IN_BYTES;
-							 break;
-
-						case 'mb':
-						case 'MiB':
-							$divisor = MB_IN_BYTES;
-							break;
-
-						case 'kb':
-						case 'KiB':
-							$divisor = KB_IN_BYTES;
-							break;
-
-						case 'b':
-						case 'B':
-						default:
-							$divisor = 1;
-							break;
-					}
+				switch ( $size_format ) {
+					case 'TB':
+							$divisor = pow( 1000, 4 );
+						break;
+
+					case 'GB':
+							$divisor = pow( 1000, 3 );
+						break;
+
+					case 'MB':
+							$divisor = pow( 1000, 2 );
+						break;
+
+					case 'KB':
+							$divisor = 1000;
```

Comment:
Indentation still off:
```suggestion
						$divisor = 1000;
```